src/hearings/containers/view-hearing/view-hearing.component.ts

```diff
@@ -1,15 +1,27 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { LocationSearchContainerComponent } from '../request-hearing/location-search-container/location-search-container.component';
 
 @Component({
   selector: 'exui-view-hearing',
   templateUrl: './view-hearing.component.html',
   styleUrls: ['./view-hearing.component.scss']
 })
-export class ViewHearingComponent implements OnInit {
-
-  constructor() { }
-
-  public ngOnInit(): void {
+export class ViewHearingComponent {
+  @ViewChild(LocationSearchContainerComponent) public searchLocationContainerComponent: LocationSearchContainerComponent;
+  public findLocationFormGroup: FormGroup;
+  dirty: boolean;
+  constructor(fb: FormBuilder) { 
+    this.findLocationFormGroup =  fb.group({
+      locationSelectedFormControl: [null, Validators.required]
+    });
   }
 
+  public getLocationSearchFocus() {
+    if (this.searchLocationContainerComponent.searchLocationComponent && 
+        this.searchLocationContainerComponent.searchLocationComponent.autoCompleteInputBox && 
```

Comment:
why there are changes on view hearing component?

Reply:
I have to do validation for the summary